### PR TITLE
Fix #355: change use cases

### LINKED ISSUE #355: Rename Use Cases section ?
In the editors calls @mlagally raised concerns that the current "[Use Cases](https://w3c.github.io/wot-scripting-api/#use-cases)" section might conflict with the [use-cases document](https://w3c.github.io/wot-usecases/).

There were proposals floating around to rename it to something like "Scenarios" (that require certain capabilities)

Any thoughts?

### PR DESCRIPTION
**Pull request by zolkis**
Signed-off-by: Zoltan Kis <zoltan.kis@intel.com>

***
<a href="https://pr-preview.s3.amazonaws.com/zolkis/wot-scripting-api/pull/375.html" title="Last updated on Feb 14, 2022, 7:04 PM UTC (560d10c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-scripting-api/375/f234394...zolkis:560d10c.html" title="Last updated on Feb 14, 2022, 7:04 PM UTC (560d10c)">Diff</a>

### CODE REVIEW

**Comment by danielpeintner** [comment]
fixes https://github.com/w3c/wot-scripting-api/issues/355

**Review by danielpeintner — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -224,9 +230,11 @@
     -->
   </section>
 
-  <section class="informative"> <h3>Use Cases</h3>
+  <section class="informative"> <h3>Scripting Use Cases</h3>
     <p>
-      The following scripting use cases are supported in this specification:
+      The business use cases listed in the [[WOT-USE-CASES]] document may be
+      implemented using this API. The following scripting use case scenarios are
+      supported in this specification.
```
Comment (relu91) [inline index.html:237]:
```suggestion
      implemented using this API. The following describes scripting use case scenarios
      supported in this specification.
```

What about having the final sentence in an active form?

**Review by relu91 — APPROVED** [review]
LGTM, just a minor comment below:

File: index.html (hunk 1 above)
Comment (danielpeintner) [reply index.html:237]: either way is fine with me

Comment (danielpeintner) [reply index.html:237]: @ashimura prefers the following title "Scripting Use Case Scenarios"

**Comment by danielpeintner** [comment]
@zolkis are you ok with the proposal from Kaz? "Scripting Use Case Scenarios"

**Comment by danielpeintner** [comment]
Today's Scripting call
@zolkis will update PR and we can merge

**Review by relu91 — APPROVED** [review]

**Comment by danielpeintner** [comment]
As decided in the call I merge the PR.

Thanks @zolkis !

**Pull request merged**